python/CreatePipWhl.cmake

```diff
@@ -36,10 +36,10 @@ if (PYTHON)
     file(MAKE_DIRECTORY ${CMAKE_BINARY_DIR}/python/ngraph_bridge)
 
     # Get the list of libraries we need for the Python pip package
-    # If we are building on CentOS then it's lib64 - else lib
+    # If we are building on a 64-bits OS then it's lib64 - else lib
```

Comment:
Is this statement accurate? Ubuntu is also 64bit OS - no?

Reply:
@avijit-nervana I've tested on Open SUSE

Reply:
@samolisov Sure. This sounds like a good rewording.

Reply:
@avijit-nervana Well, since the master has been uploaded to my branch and there are some merging conflicts during rebasing, I've made a new commit to the change.